### DISCUSSION TITLE
Multiple user editing

### DISCUSSION THREAD

**Discussion by ahmadlimbada** [GrapesJS/grapesjs#4048] [Ideas]
It will be such a great feature. if multiple user can work on same webpage while using the remote server. Just like google docs multi-user editing on same webpage would be such a nice feature.

**Comment by Ju99ernaut** [comment]
Probably something that's planned, it's been mentioned here https://github.com/artf/grapesjs/pull/2524. Have tried this using firebase in the past, kinda worked but it's a bit inefficient when it comes to refreshing the canvas with the current API.

**Reply by ahmadlimbada** [reply]
I am working on the same in MERN Stack. But the issue is onload canvas get refreshed and user lost unsaved work.
